packages/output/src/index.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {Writable} from 'stream';
+
+type Callback = (error: Error | null | undefined) => void;
+type Encoding = string | undefined;
+
+class Output {
+  private _stdout: NodeJS.WriteStream;
+  private _stderr: NodeJS.WriteStream;
+
+  constructor() {
+    this._stdout = process.stdout;
+    this._stderr = process.stderr;
+  }
+
+  get isTTY() {
+    return (this._stdout as NodeJS.WriteStream).isTTY;
```

Comment:
Why do we need to cast this?

Reply:
Since we are casting the stream in `setErrorStream` and `setOutStream`, turns out we don't. Removed it now 👍